pytest/lib/messages/network.py

```diff
@@ -64,6 +64,22 @@ class RoutedMessage:
     pass
 
 
+class ClosingReason:
+    pass
+
+
+class PoolError:
+    pass
+
+
+class ReasonForBan:
+    pass
+
+
+class RegisterPeerError:
+    pass
+
+
```

Comment:
do you really need to touch the python code? Please try to avoid replicating the rust network code in python code. Implement relevant tests in rust instead.